relational_learner/episodes_to_novelty episodes
bool visualise_graphs
---
float64[] spatial_nov
float64[] temporal_nov
float64 roi_knowledge
